web-common/src/components/virtualized-table/core/ColumnHeader.svelte

```diff
@@ -25,8 +26,11 @@
   export let header;
   export let position: HeaderPosition = "top";
   export let enableResize = true;
+  // FIXME: pretty sure isSelected can be deprecated in favor of highlight
   export let isSelected = false;
   export let bgClass = "";
+  export let highlight: boolean;
```

Comment:
This seems to be unused. Expected?

Reply:
thanks, fixed this -- that was supposed to be used but apparently I didn't finish that cleanup!!